### ISSUE TITLE
Registration does not work

### ISSUE THREAD

**Issue by F3nd0** [nickolas360/harmony#9]
Sorry if this is a duplicate of #8, but I got a different error. When I try to register an account, I get the following message:

```
Registration failed. Errors:
  email:
    Please update Discord to continue.
```

This has happened with two different e-mail providers (@seznam.cz and @posteo.ru). I also tried a third one, which, for a change, led me to this error:

```
Registration failed. Errors: {
  "code": 0,
  "message": "500: Internal Server Error"
}
```

I'd be happy to do more tests or try to get more info, if that helps. Thank you!

**Comment by F3nd0** [comment]
As a brief update, tried changing `v6` to `v7` in the API link in `discord.py`, as well as updating the user agents in `user_agents.py` (thanks to @polarisfm for the tip), but to little avail. It still wants me to update Discord.

**Comment by nickolas360** [comment]
I believe I've fixed this issue in version 0.6.0—I was able to register successfully. Unfortunately, issues with reCAPTCHA are often making it impossible to test. Please see [this issue for librecaptcha](https://github.com/nickolas360/librecaptcha/issues/7).

**Comment by F3nd0** [comment]
Worked like a charm. Thank you very much for the fix!
